src/Uno.UI/UI/Xaml/Controls/Primitives/ButtonBase/ButtonBase.Android.cs

```diff
@@ -65,9 +65,18 @@ protected override void OnUnloaded()
 							this.Log().Debug("TouchUpInside, executing command");
 						}
 
-						// TODO: Simulate the complete pointer sequence on "this", and remove the Tapped and Click
-						// uiControl.SetOnTouchListener()
-						OnPointerPressed(new PointerRoutedEventArgs(this));
+						var downTime = Android.OS.SystemClock.UptimeMillis();
+						var eventTime = downTime;
+						var evt1 = MotionEvent.Obtain(
```

Comment:
hmm... there's no way to get the actual `MotionEvent`?

Reply:
This code is actually completely removed. Click can be handled completely using pointer events, and not the native Click event.